source/_integrations/bang_olufsen.markdown

```diff
@@ -250,6 +252,31 @@ data:
 
 WebSocket notifications received from the device are fired as events in Home Assistant. These can be received by listening to `bang_olufsen_websocket_event` event types, where `device_id` or `serial_number` can be used to differentiate devices.
 
+### Button events
```

Comment:
Let's remove button from here. While I get the idea from the integrated device perspective, from the Home Assistant end naming it buttons can be confusing, as that is a different concept.


```suggestion
### Events
```